src/action-inputs.ts

```diff
@@ -8,7 +8,7 @@ export const getInputs = (): IActionInputs => {
   const BRANCH_IGNORE_PATTERN: string = core.getInput('skip-branches', { required: false }) || '';
   const CUSTOM_ISSUE_NUMBER_REGEXP = core.getInput('custom-issue-number-regexp', { required: false });
   const JIRA_PROJECT_KEY = core.getInput('jira-project-key', { required: false });
-
+  const FAIL_PR_WHEN_JIRA_ISSUE_NOT_FOUND = core.getInput('fail-pr-when-jira-issue-not-found', { required: false }) || 'false';
```

Comment:
need to cast ti to boolean and use boolean value in the interface

Reply:
Done